utils/index.ts

```diff
@@ -0,0 +1,194 @@
+export const random = (min: number, max: number) => {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+};
+
+//grid arrays/shapes
+export const gridDefault = (): number[] | number[][] => {
+  const rows = 18;
+  const cols = 10;
+  var array = [];
+  const emptyArray: number[] = [];
+
+  // Fill array with 18 arrays each containing
+  // 10 zeros (0)
+  for (let row = 0; row < rows; row++) {
+    array.push(emptyArray);
+    for (let col = 0; col < cols; col++) {
+      array[row].push(0);
+    }
+  }
+
+  return array;
+};
+
+// Random Shape
+export const randomShape = () => {
+  return random(1, shapes.length - 1);
+};
+
+// Define block shapes and their rotations as arrays.
+export const shapes: [number, number, number, number][][][] = [
```

Comment:
This is the array where all the shapes are defined and their rotated positions as well.. They each are 4x4 grid. 0 represents the blank background while 1 should be the colored part